src/aspire/apple/apple.py

```diff
@@ -40,14 +88,33 @@ def __init__(self, output_dir=None):
             self.query_image_size = query_image_size
```

Comment:
This code (83-88) only runs if the user explicitly sets `query_image_size=None`. The default value, `52` is 2/3 of the original default particle size, `78`, but the `particle_size` is now a mandatory argument. If the user didn't know better, they would be setting `query_image_size` to 52 regardless of `particle_size`. Perhaps `query_image_size` should default to `None`?

Reply:
yes, that would be bad, good catch.